layouts/_default/index.html

```diff
@@ -25,12 +25,12 @@ <h2 class="post-title">
             <time class="post-date">
               {{- $date := .Date | time -}}
```

Comment:
`time` is redundant here (and in other places), because we are calling it later anyway. This should be just:

```
{{- $date := .Date -}}
```